hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/S3AFileSystem.java

```diff
@@ -330,6 +331,11 @@
    */
   private DirectoryPolicy directoryPolicy;
 
+  /**
+   * Header processing for XAttr. Created on demand.
```

Comment:
nit: is "Created on demand" accurate? it is created in `initialize`.

Reply:
you are right. I originally had it on demand for XAttr calls, but once I pulled in to the copy operation just made it something created earlier.